Dockerfile

```diff
@@ -0,0 +1,124 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+WORKDIR /build
+
+COPY . /build/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        autoconf \
+        automake \
+        bzip2-dev \
+        curl-dev \
+        curl-dev \
+        file \
+        fts-dev \
+        g++ \
+        json-c-dev \
+        libmilter-dev \
```

Comment:
Unless running clamav-milter in the docker enviroment is a part of the plan, you may wish to remove this and disable milter (for CMake `-D ENABLE_MILTER=OFF`).

Reply:
I think it depends what the users of the container want, right? The choice is defined in the init script however